pipeline/normalize.py

```diff
@@ -31,12 +31,21 @@ def strip_whitespace(df, col):
     df[col] = df[col].str.strip()
     return df
 
+def negative_lng(df):
```

Comment:
This function is defined, but never used, so i think if you run this on the data, it won't change the longitudes. I would try it out to make sure it's doing what you expect.